### DISCUSSION TITLE
Network Graph Filtering

### DISCUSSION THREAD

**Discussion by cruznunez** [community/community#9193] [Pull Requests]
When reviewing PRs, I sometimes check the network graph to see if we aren't including any changes we may not want. I think it would be nice to be able to toggle visibility for each of the branches because sometimes there's a lot of branches going on and it can get confusing to just look at the 2 or 3 branches that I want. Example below.

![vm_695](https://user-images.githubusercontent.com/8742021/146996166-c5f5b175-959d-40fc-b184-b543b22d1aa9.jpg)

I really only need to see the `dev` branch, the `release/1.26.0` branch, and the `yancy/import-csv-not-working/VM-695` branch. If all the other branches were hidden, that would be great.

Also, separate issue but changing the scale on the graph so that we can see more or less of the branches would be great in this scenario too.

**Comment by cruznunez** [comment]
1 month and no response :'c

**Comment by cruznunez** [comment]
2 months and no response :''c

**Comment by kirkbushell** [comment]
This would be amazing.
